src/wp-includes/script-loader.php

```diff
@@ -1223,15 +1222,13 @@ function wp_default_scripts( $scripts ) {
 			'invalidDate'             => __( 'Invalid date.' ),
 			'invalidValue'            => __( 'Invalid value.' ),
 			'blockThemeNotification'  => sprintf(
-				/* translators: 1. %s: URL to the localized version of https://wordpress.org/support/article/site-editor/, 2: HTML Button. */
-				__( 'Hurray! Your theme supports Full Site Editing with blocks. <a href="%1$s" target="_blank">Tell me more</a>. %2$s' ),
+				/* translators: 1: HelpHub link to Site Editor documentation, 2: HTML Button. */
+				__( 'Hurray! Your theme supports Full Site Editing with blocks. <a href="%1$s">Tell me more</a>. %2$s' ),
 				__( 'https://wordpress.org/support/article/site-editor/' ),
 				sprintf(
-					/* translators: 1: URL to the Site Editor admin screen, 2: Button label, 3: Button text. */
```

Comment:
Shouldn't this translators comment be left in (with `3: Button text` removed)?

Reply:
D'oh!  of course not, my bad.

Reply:
👍  :)